website/src/content/api-reference/custom-webpack-config.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Custom Webpack Config"
+---
+
+<Subtitle>
+The utilities allows you create a custom webpack configuration for a Merchant Center Application.
+</Subtitle> 
+
+# Installation
+
+```
+yarn add @commercetools-frontend/mc-scripts
+
+# or
+
+npm --save install @commercetools-frontend/mc-scripts
+```
+
+# Usage
+To create a custom webpack configuration for a Merchant Center application, you can use `createWebpackConfigForProduction` for production and `createWebpackConfigForDevelopment` for development.
+These are factory functions used to create a default webpack config. They both accept some optional arguments.
+
+## Production
+For production mode, you should create a `webpack.config.prod.(cjs, js)` file and use the function `createWebpackConfigForProduction` to create a default webpack config.
+
+```tsx title="webpack.config.prod.cjs"
+const { createWebpackConfigForProduction } = require("@commercetools-frontend/mc-scripts");
+
+const config = createWebpackConfigForProduction(options: TWebpackConfigOptions);
+
+// you can include your webpack config by updating the default config object
+config.devtool: 'cheap-source-map'
+
+module.exports = config; 
+```
+
+### options
```

Comment:
Since most of the options are the same, I would move this into its own section on the page. I would also document only the more relevant and useful options, as some of them are mostly used internally.

So the structure of the page would be like:

```
(intro)

# Usage

# For development

# For production

# Options

## Including additional source folders
```

So in the `Options` section I would actually only document the `sourceFolders` option, as this is the most relevant one for users.

I think it would be better to use subsections so that we can describe the use case and provide examples. For `sourceFolders` I would suggest to name the section `Including additional source folders`.

If you need help with the content, let me know.